.circleci/config.yml

```diff
@@ -141,7 +146,8 @@ workflows:
               ignore: /.*/
   unit-test-workflow:
     jobs:
-      - commit-build-test
+      - commit-build-test:
+          context: terraform provider
```

Comment:
Why are we adding the context in the unit test, and will this work for everyone?
this is a restricted context, and users outside of the organisation cannot use it. Could we test it by creating a fork from a user outside of the organisation and raising a PR from that fork?

Reply:
We need to pass an env var called SONAR_TOKEN to be able to send the coverage to SonarCloud.

Reply:
We will need a new context for this token (also see discussion on PD). This will allow people to run arbitrary code using our creds from outside the organisation.

Reply:
No, we will need to put it into the project environment (if we come to the decision of allowing the use of the token by anyone). Restricted contexts can only be used by organisation members - see [here](https://circleci.com/docs/2.0/contexts/)
> To use environment variables set on the Contexts page, the person running the workflow must be a member of the organization for which the context is set.